hercules/plant_components/wind_farm.py

```diff
@@ -124,28 +161,274 @@ def __init__(self, h_dict):
                 f"in the wind input file ({max_time})"
             )
 
-        # Set starttime_utc (zero_time_utc is redundant since time=0 corresponds to starttime_utc)
+        # Set starttime_utc
         self.starttime_utc = starttime_utc
 
         # Determine the dt implied by the weather file
-        self.dt_wi = df_wi["time"][1] - df_wi["time"][0]
+        self.dt_wi = df_wi["time"].iloc[1] - df_wi["time"].iloc[0]
 
         # Log the values
         if self.verbose:
             self.logger.info(f"dt_wi = {self.dt_wi}")
             self.logger.info(f"dt = {self.dt}")
 
+        self.logger.info("Interpolating wind input file...")
+
         # Interpolate df_wi on to the time steps
         time_steps_all = np.arange(self.starttime, self.endtime, self.dt)
         df_wi = interpolate_df(df_wi, time_steps_all)
 
-        # FLORIS PREPARATION
+        # INITIALIZE FLORIS BASED ON WAKE MODEL
+        if self.wake_model == "precomputed":
+            self._init_floris_precomputed(df_wi)
+        elif self.wake_model == "dynamic":
+            self._init_floris_dynamic(df_wi)
+        else:  # wake_model == "no_added_wakes"
+            self._init_floris_none(df_wi)
```

Comment:
`elif self.wake_model == "no_added_wakes"` seems more robust here, but not a big deal unless the user reaches in and sets `self.wake_model` to something else, which seems pretty unlikely

Reply:
Leaving as is, not critical.